### ISSUE TITLE
Boiling Blood quest broken after 4ee64a232ad

### ISSUE THREAD

**Issue by Aokromes** [TrinityCore/TrinityCore#28825]
### Description

you get no target error after  https://github.com/TrinityCore/TrinityCore/commit/4ee64a232ad830255bc6779e595d412c934b6b74

### Expected behaviour

quest works

### Steps to reproduce the problem

take quest 10538 
get items
try to use on cauldron
you get no target error

### Branch

3.3.5

### TC rev. hash/commit

4ee64a232ad830255bc6779e595d412c934b6b74

### Operating system

n/r

### Custom changes

n/r

**Comment by Krutok** [comment]
use: UPDATE `gameobject_template` SET `Data8` = 0 WHERE `entry` = 184722;

Restart Server and Quest works for me

**Comment by MaxtorCoder** [comment]
You don't touch `gameobject_template` as this is sniffed.

**Comment by Krutok** [comment]
> You don't touch `gameobject_template` as this is sniffed.

But that's how the quest works then I just tested it

**Comment by Shauren** [comment]
This trap gameobject is not supposed to be visible to players so the fix is wrong

**Comment by Aokromes** [comment]
> > You don't touch `gameobject_template` as this is sniffed.
> 
> But that's how the quest works then I just tested it

it's incorrect fix, the correct fix is at https://github.com/TrinityCore/TrinityCore/pull/28827

**Comment by Krutok** [comment]
is fixed in master not in 3.3.5a, because otherwise the bug wouldn't exist if it were fixed

**Comment by MaxtorCoder** [comment]
> This trap gameobject is not supposed to be visible to players so the fix is wrong
> 
> @MaxtorCoder trap gameobjects arent sent to client and we only got data fields 0-7 from gameobject.db2

Ah fair, I did not know traps aren't sent :)

**Comment by Aokromes** [comment]
> is fixed in master not in 3.3.5a, because otherwise the bug wouldn't exist if it were fixed

it's bugged also on master, it needs to be tested and merged on both branches.

**Comment by Krutok** [comment]
> > is fixed in master not in 3.3.5a, because otherwise the bug wouldn't exist if it were fixed
> 
> it's bugged also on master, it needs to be tested and merged on both branches.

Yes, but until then just set to 0 as a temporary solution and you can do the quest: https://www.youtube.com/watch?v=FiQV00tew_w

**Comment by Nyr97** [comment]
Temporary solutions aren't the best, especially with these stuff which might be bugged for other quests.

**Comment by CraftedRO** [comment]
I'm just curious to see how long will take for that commit to be "cherry-picked" to 3.3.5 too.

**Comment by meji46** [comment]
https://github.com/TrinityCore/TrinityCore/commit/19ea4c416aa1f80798739546f6fd2e63699775d2

**Comment by anyanfei** [comment]
[Aokromes](https://github.com/Aokromes) thanks , i'm also looking for a fix for this quest , nice to see you again :-)

**Comment by CraftedRO** [comment]
@anyanfei the quest works on latest tc revision, update your core.

**Comment by anyanfei** [comment]
> @anyanfei the quest works on latest tc revision, update your core.

thanks ,bro, nice to see you again at here :-)
